src/_typed_dict_test_helper.py

```diff
@@ -16,7 +16,7 @@ class Foo(TypedDict):
 
 
 class FooGeneric(TypedDict, Generic[T]):
-    a: Optional[T]
+    a: Optional[T]  # noqa: UP045 while < (3, 10)
```

Comment:
UP045 was previously part of UP007, but was split into its own rule in a recent release. We have UP007 ignored for the whole repo; I think it probably makes sense to have UP045 ignored for the whole repo as well TBH